ClubManagement/ClubManagement/Controllers/FirebaseController.cs

```
@@ -19,32 +21,50 @@ protected FirebaseController()
 
         public async void Add(T t)
         {
-            await FirebaseClient.PostAsync(t);
+            try
+            {
+                await FirebaseClient.PostAsync(t);
+            }
+            catch { }
         }
 
         public List<T> Values
         {
             get
             {
-                var users = FirebaseClient.OnceAsync<T>().Result;
-
-                return users.Select(x =>
+                try
                 {
-                    x.Object.Id = x.Key;
+                    var users = FirebaseClient.OnceAsync<T>().Result.Select(x =>
+                    {
+                        x.Object.Id = x.Key;
 
-                    return x.Object;
-                }).ToList();
+                        return x.Object;
+                    }).ToList();
+                    return users;
+                }
+                catch (Exception)
```

This will hide the error and user can't see the error